openquake/calculators/tests/classical_test.py

```diff
@@ -165,6 +165,7 @@ def test_case_14(self):
 
     @attr('qa', 'hazard', 'classical')
     def test_case_15(self):  # full enumeration
+        check_platform('xenial', 'trusty')
```

Comment:
I have moved down the `check_platform` line, so now the CSV are tested on any platform. Only the .geojson and .xml tests are skipped since they rely on the exact numbers (the tolerance feature is implemented only for CSV).